Apps/Common/src/Authorization/UserAuthorizationHandler.cs

```diff
@@ -24,18 +24,16 @@ protected override Task HandleRequirementAsync(AuthorizationHandlerContext conte
         {
             string hdidClaim = context?.User.FindFirst(c => c.Type == "hdid").Value;
 
-            // We custom map the subject id to a custom hdid claim inside the JWT, so we need to check that it
-            // matches what is being used for the target subject
             if (!string.Equals(hdidClaim, hdid, System.StringComparison.Ordinal))
             {
                 #pragma warning disable CA1303 // Do not pass literals as localized parameters
-                System.Console.WriteLine(@"hdid matches JWT");
+                System.Console.WriteLine(@"hdid parameter doest not match user's JWT");
```

Comment:
This should be using a logger instead of writing to System.Console.

Reply:
fixed.